test/extended/authentication/ldap/rfc2307/sync-config-partially-user-defined.yaml

```diff
@@ -18,6 +18,5 @@ rfc2307:
     baseDN: "ou=people,ou=rfc2307,dc=example,dc=com"
     scope: sub
     derefAliases: never
-    filter: (objectclass=inetOrgPerson)
```

Comment:
this seems wrong... for rfc2307, aren't we still looking up all users based on the filter, then extracting the user id attribute by looking at the dn here?